core-rules/src/main/java/org/hisp/dhis/rules/RuleVariableValue.java

```diff
@@ -47,7 +47,7 @@ static RuleVariableValue create(@Nonnull String value,
     }
 
     @Nonnull
-    static RuleVariableValue create(@Nonnull String value,
+    public static RuleVariableValue create(@Nonnull String value,
```

Comment:
We should not change this method to public because It's [private to package](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). The goal is not to access from outside from package to this class,  only RuleEngine, RuleEngineContext ... can access it. If we change it to public we are breaking encapsulation.